### ISSUE TITLE
Overlapping nodes, async task node size not determined properly?

### ISSUE THREAD

**Issue by geordiemhall** [fpwong/BlueprintAssistWiki#99]
Have been trying out BlueprintAssist and mostly loving the auto-formatting, though just ran into a case where it seemingly couldn't determine the size of this (async task) node properly, and so let it overlap with wires and other nodes.

![image](https://user-images.githubusercontent.com/3511383/201803807-3faa802a-46fd-4ea3-b1dc-b80ff69b1419.png)

Not sure if there's something specific about async tasks nodes, since other nodes with variable number of pins, or pins that can cause the node to get wider/taller with wrapping seem to be handled okay in general.

Thanks!

**Comment by fpwong** [comment]
Hi @geordiemhall, thanks just tested this and the plugin is failing to detect changes in the FText pins. Will fix this in the next update.

As a temporary fix you can manually refresh the node size by selecting that node and pressing `CTRL + SHIFT + R`.

**Comment by geordiemhall** [comment]
Hi @fpwong thanks for the quick reply! I tried using `Ctrl + Shift + R` (and double checked it was still bound to 'Update Node Sizes', but nothing happened). Also tried turning on 'Slow but accurate node sizing', but still nothing. 

Though found that if I disconnected an exec, then did format all `Ctrl + R`, then reconnected it and `Ctrl + R` again it had cached the proper sizes.

But yeah seems like maybe Text pins driven by string table entries (whose text can change without you touching the node) might need special handling 👍
